recipes/apr/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "apr" %}
+{% set version = "1.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://archive.apache.org/dist/{{ name }}/{{ name }}-{{ version }}.tar.bz2
+  sha256: 7d03ed29c22a7152be45b8e50431063736df9e1daa1ddf93f6a547ba7a28f67a
+  patches:
+    # Skip a failing test on Mac due to `poll`'s behavior.
+    #
+    # ref: https://www.mail-archive.com/dev@apr.apache.org/msg26625.html
+    #
+    - macos_testpoll.c.patch  # [osx]
+    # Apply a patch to fix a double free issue on Windows.
+    #
+    # ref: https://bz.apache.org/bugzilla/show_bug.cgi?id=59711
+    #
+    - win_apr_app.c.patch     # [win]
```

Comment:
Needed this patch to fix a double free issue on Windows. This also caused a test failure. More details in the bug report linked below.

ref: https://bz.apache.org/bugzilla/show_bug.cgi?id=59711